cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/statements/LabelStatement.kt

```diff
@@ -48,6 +50,14 @@ class LabelStatement : Statement() {
             .toString()
     }
 
+    override var statementEdges: MutableList<PropertyEdge<Statement>>
+        get() =
+            subStatement?.let { mutableListOf(PropertyEdge(this, it, mutableMapOf())) }
+                ?: mutableListOf()
+        set(value) {
+            subStatement = value.firstOrNull()?.end
+        }
+
```

Comment:
I would prefer using the `wrap` / `unwrap` function instead of doing it manually, since they already do "some" magic and might do more magic in the future.

```suggestion
override var statementEdges: MutableList<PropertyEdge<Statement>>
        get() {
            return subStatement?.let { wrap(listOf(it), this) } ?: mutableListOf()
        }
        set(value) {
            subStatement = unwrap(value).firstOrNull()
        }
```